Cargo.toml

```diff
@@ -29,7 +29,8 @@ default-features = false
 features = ["secure", "use-bindgen"]
 
 [build-dependencies]
-protobuf-build = { version = "0.12", default-features = false }
+# protobuf-build = { version = "0.12", default-features = false }
+protobuf-build = { git = "https://github.com/hicqu/protobuf-build", branch = "include-google", default-features = false }
```

Comment:
Is this a pr ready to review? I don't know why we depend on a personal fork. It looks brittle.